rmf_site_editor/src/interaction/camera_controls/mod.rs

```diff
@@ -382,6 +407,11 @@ impl FromWorld for CameraControls {
             .push_children(&orthographic_child_cameras)
             .id();
 
+        let mut ambient_light = world.get_resource_mut::<AmbientLight>().expect(
+            "Make sure bevy's PbrPlugin is initialized before the cameras");
+
+        ambient_light.brightness = 2.0;
```

Comment:
Since we made everything a lot darker than default bevy, I also had to reduce the ambient light otherwise its effect would be overwhelming the rendering and everything would look washed out.